tests/jenkins/run_test_installation_docs.sh

```diff
@@ -298,17 +298,20 @@ LINUX_PYTHON_GPU_END_LINENO=$(grep -n "END - Linux Python GPU Installation Instr
 
 set_instruction_set ${LINUX_PYTHON_GPU_START_LINENO} ${LINUX_PYTHON_GPU_END_LINENO}
 
+
+# mxnet/base-cuda9 is a simple Docker Image with 'nvidia/cuda:9.0-cudnn7-devel' and 'apt-get install sudo'.
+
 echo
 echo "### Testing Virtualenv ###"
 echo "${virtualenv_commands}"
 echo
-nvidia-docker run --rm nvidia/cuda:9.0-cudnn7-devel bash -c "${virtualenv_commands}"
+nvidia-docker run --rm mxnet/base-cuda9 bash -c "${virtualenv_commands}"
```

Comment:
I think it would be more maintainable maybe with a different name as @gautamkmr  if we push this docker image for nightly tests to the docker registry. The release doesn't have to block on this though.

Reply:
Where can I see the contents of this image? Also it would be good to keep consistency with nvidia images (I assume they are taken as base anyway) and name and branch them with minor versions of cuda: ```mxnet/base:cuda9.0``` ```mxnet/base:cuda9.1```